include/llvm/CodeGen/AsmPrinter.h

```diff
@@ -634,8 +632,9 @@ class AsmPrinter : public MachineFunctionPass {
                           bool isCtor);
   GCMetadataPrinter *GetOrCreateGCPrinter(GCStrategy &C);
   /// Emit GlobalAlias or GlobalIFunc.
-  void emitGlobalIndirectSymbol(Module &M,
-                                const GlobalIndirectSymbol& GIS);
+  void emitGlobalIndirectSymbol(Module &M, const GlobalIndirectSymbol &GIS);
```

Comment:
It's generally frowned on to modify other surrounding lines, even when they're incorrectly formatted. It requires stepping through an extra commit in `git blame`. Not a big deal at all though.